pvlib/test/test_surfrad.py

```diff
@@ -0,0 +1,48 @@
+import inspect
+import os
+
+from pandas import Timestamp, DatetimeIndex
+from pandas.util.testing import network
+
+from pvlib.iotools import surfrad
+
+test_dir = os.path.dirname(
+    os.path.abspath(inspect.getfile(inspect.currentframe())))
+testfile = os.path.join(test_dir, '../data/surfrad-slv16001.dat')
+network_testfile = ('ftp://aftp.cmdl.noaa.gov/data/radiation/surfrad/'
+                    'Alamosa_CO/2016/slv16001.dat')
+
+
+def test_read_surfrad():
+    surfrad.read_surfrad(testfile)
+
+
+@network
+def test_read_surfrad_network():
+    surfrad.read_surfrad(network_testfile)
+
+
+def test_read_surfrad_columns_exist():
+    data, _ = surfrad.read_surfrad(testfile)
+    assert 'zen' in data.columns
+    assert 'temp' in data.columns
+    assert 'par' in data.columns
+    assert 'pressure' in data.columns
+
+
+def test_format_index():
+    start = Timestamp('20160101 00:00')
+    expected_index = DatetimeIndex(start=start, periods=1440, freq='1min')
```

Comment:
I think you can supply `tz='UTC'` here and skip the localize call below. Seems a little more straightforward to me. I had to look up if localize modifies in place or not.

Reply:
Good catch, thanks. This was a naive "fix" when I was reading the test failure incorrectly, I wasn't setting the timezone correctly in read_surfrad.
It looks like pandas 0.14.0 has some issues with setting the timezone the way you proposed though, so i'll have to figure that out.